packages/editor/src/components/post-saved-state/test/index.js

```diff
@@ -34,6 +36,10 @@ jest.mock( '@wordpress/compose/src/hooks/use-viewport-match', () => {
 	return mock;
 } );
 
+jest.mock( '@wordpress/icons/src/icon', () => () => (
+	<div data-testid="test-icon" />
+) );
```

Comment:
What the the issue preventing us from using the real `Icon` component?

Reply:
I didn't find a good way to query for the icon. Any recommendations?

Reply:
Mhh, I can't find either. Ideally we could at least assign a default `role="presentation"` to these icons, or at otherwise label them with some accessible text. But it's not a topic for this PR

Reply:
Also, [the `Button` in `PostSavedDate`](https://github.com/WordPress/gutenberg/blob/81e8e0f7c999df3cc43ef97cb2913779c0190eff/packages/editor/src/components/post-saved-state/index.js#L138-L162) can render 2 icons at the same time (one as `children` and one via the `icon` prop) — that's weird